tests/filecheck/cf_ops.xdsl

```diff
@@ -34,7 +34,7 @@ module() {
   ^4(%c : !i1, %a: !i32):
     cf.br(%c : !i1, %a : !i32)(^5)
   ^5(%cond : !i1, %arg: !i32):
-    cf.cond_br(%cond: !i1, %cond: !i1, %arg : !i32, %arg : !i32, %arg : !i32, %arg : !i32)(^5, ^6) ["operand_segment_sizes" = !dense<!vector<[3 : !i64], !i32>, [1 : !i32, 2 : !i32, 3 : !i32]>]
+    cf.cond_br(%cond: !i1, %cond: !i1, %arg : !i32, %arg : !i32, %arg : !i32, %arg : !i32)(^5, ^6) ["operand_segment_sizes" = !dense<!vector<[2 : !i64], !i32>, [2 : !i32, 3 : !i32]>]
```

Comment:
Actually not this one!
When cleaning IRDL, I also fixed the bug that non-variadics operands/results are appearing in the segment sizes. I preferred having everything in the same commit, since breaking this in two commits is a bit painful.